.github/workflows/publish-sdist.yml

```diff
@@ -10,9 +10,9 @@ jobs:
     name: Build and publish on PyPI
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@master
```

Comment:
Although it can require manual updates, it seems safer to me to specify the `@vX` version, so it will be possible to review the changes before upgrading.
Side note: it seems that v2 and v3 versions of actions/checkout are both maintained in https://github.com/actions/checkout/releases